sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -262,9 +262,9 @@ class annotated_ptr {
     using reference = annotated_ref<T, PropertyListT>;
 
     annotated_ptr() noexcept;
-    annotated_ptr(T *Ptr, const PropertyListT &P = properties{}) noexcept;
+    explicit annotated_ptr(T *Ptr, const PropertyListT &P = properties{}) noexcept;
```

Comment:
Correct. What's still valid is:
```
annotated_ptr<int, decltype<properties{}>> x(p);
```
or
```
auto x = annotated_ptr<int, decltype<properties{}>(p);
```

Reply:
It seems I missed this. This will prevent user from passing a raw pointer to a kernel struct whose member is annotated_ptr. Not sure if we really want that. We can talk about it tomorrow